network/p2p/utils/ratelimiter/rate_limiter.go

```diff
@@ -39,7 +39,7 @@ func NewRateLimiter(limit rate.Limit, burst int, lockoutDuration time.Duration,
 		limiterMap:               internal.NewLimiterMap(rateLimiterTTL, cleanUpTickInterval),
 		limit:                    limit,
 		burst:                    burst,
-		rateLimitLockoutDuration: lockoutDuration * time.Second,
+		rateLimitLockoutDuration: lockoutDuration,
```

Comment:
can you split these fixes out into their own PR. this looks like the rate limits currently have a permanent lockout